### PR TITLE
Rename

### PR DESCRIPTION
**Pull request by irisslee**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -11,32 +11,30 @@ Installation
...
-      GeoMAC. Returns GeoJSON.
+      NIFC. Returns GeoJSON.
 
     Options:
       --help  Show this message and exit.
 
     Commands:
-      active-perimeters   Perimeters of active fires in a recent 24-hour period
-      all-perimeters      Perimeters of all fires
+      current-perimeters   Perimeters of active fires
```
Comment (palewire) [inline README.rst]: Is there a difference between being current and being active?

File: nifc_wildfires/__init__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+import requests
+
+
+def get_current_perimeters():
+    """
+    Get geojson fire perimeters from new 2021 feed.
```
Comment (palewire) [inline nifc_wildfires/__init__.py]: Can we rewrite this to be more descriptive of what we're pulling and less about being "new"?

File: nifc_wildfires/__init__.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+    Returns a GeoJSON object.
+    """
+    r = requests.get("https://opendata.arcgis.com/datasets/2191f997056547bd9dc530ab9866ab61_0.geojson")
+    if r.status_code != 200:
+        raise Exception(f"Request for data failed with {r.status_code} status code")
+    return r.json()
+
+
+def get_nifc_incidents():
```
Comment (palewire) [inline nifc_wildfires/__init__.py]: If we're renaming the library nifc, I think we can skip calling the functions nifc too. What if this was just `get_incidents` for simplicity's sake?

File: README.rst (hunk 1 above)
Comment (irisslee) [reply README.rst]: No. which is why we are only keeping one - "current-perimeters"

Comment (palewire) [reply README.rst]: Gotcha. Does current mean the same thing as active? My only little picky thought is that we should try to use the same term across all of our repositories. For instance, if we use current here, we shouldn't use active in another one.

File: nifc_wildfires/__init__.py (hunk 3 above)
Comment (irisslee) [reply nifc_wildfires/__init__.py]: yup

File: nifc_wildfires/__init__.py (hunk 2 above)
Comment (irisslee) [reply nifc_wildfires/__init__.py]: yup

Comment (palewire) [reply nifc_wildfires/__init__.py]: Kay. Just go ahead and make the edits and push it up. I'll look again after that stuff is done.

File: README.rst (hunk 1 above)
Comment (irisslee) [reply README.rst]: ok I will stick with "Active"

File: nifc_wildfires/cli.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+@click.group()
+def cmd():
+    """
+    A command-line interface for downloading wildfire perimeter and incident points data from NIFC.
+    Returns GeoJSON.
+    """
+    pass
+
+
+@cmd.command(help="Perimeters of active fires in new 2021 feed")
```
Comment (palewire) [inline nifc_wildfires/cli.py]: Let's clip "new" out of here too and make this language a little more evergreen.

File: setup.py [hunk 5]
```diff
@@ -8,24 +8,22 @@ def read(fname):
 
 
 setup(
-    name='geomac-wildfires',
+    name='nifc-wildfires',
     version='0.1.14',
     description="Download wildfires data from NIFC",
     long_description=read('README.rst'),
     author='Los Angeles Times Data and Graphics Department',
     author_email='datagraphics@latimes.com',
```
Comment (palewire) [inline setup.py:16]: Let's update this to datagraphics@caltimes.com

**Comment by palewire** [comment]
Thank you. This looks great.

**Pull request merged**